go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cilium/ebpf
 
-go 1.23
+go 1.23.0
```

Comment:
This runs into https://github.com/cilium/ebpf/commit/eaef6a7c6a620f7e673463e367c8e3ede8c4ca57 again. I'm really quite sick of the the whole add a .0 or don't add a .0 to the go line saga xD

Reply:
I'd be grateful for some semi-official answer what the thing to do is for depending on a language version. I don't want to pin toolchain because I don't really understand why its necessary and it's another thing to manage.

Reply:
I'm also equally confused about the meaning of the `toolchain` directive on projects like these. I guess it's the toolchain version used when running `go test`, etc. Maybe we can let this be managed by dependabot, because I don't see a way of configuring this in dependabot.yml. There's no ecosystem-specific configuration whatsoever, and we don't even have a gomod block in there, it's enabled through the GitHub UI.